src/Microsoft.IdentityModel.JsonWebTokens/Json/JsonClaimSet.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.ObjectModel;
 using System.Globalization;
 using System.Security.Claims;
+using System.Text;
```

Comment:
is this used?

Reply:
Yes, when using Encoding. VS is not good at figuring out unused namespaces when `#if`s are used.